README.md

```diff
@@ -0,0 +1,9 @@
+# Some Questions
+
+1. May I know about the differences between `test1` and `test2` ,Yang?
```

Comment:
there are no diffierece, just test it to see if i can upload some codes

Reply:
Well, sounds not bad. May you merge this pull request to your repo? :-)